src/components/Insights/HeatMap.js

```diff
@@ -18,337 +18,356 @@ import 'leaflet.markercluster/dist/MarkerCluster.Default.css';
 import '../../styles/Insights/HeatMap.css';
 import 'react-progress-bar-plus/lib/progress-bar.css';
 
-const FluxMixin = Fluxxor.FluxMixin(React),
-      StoreWatchMixin = Fluxxor.StoreWatchMixin("DataStore");
+const FluxMixin = Fluxxor.FluxMixin(React);
+const StoreWatchMixin = Fluxxor.StoreWatchMixin("DataStore");
 const PARELLEL_TILE_LAYER_RENDER_LIMIT = 200;
 const SENTIMENT_FIELD = 'neg_sentiment';
 const defaultClusterSize = 40;
 const DEFAULT_LANGUAGE = 'en';
 
+const MAPBOX_ACCESS_TOKEN = 'pk.eyJ1IjoiZXJpa3NjaGxlZ2VsIiwiYSI6ImNpaHAyeTZpNjAxYzd0c200dWp4NHA2d3AifQ.5bnQcI_rqBNH0rBO0pT2yg';  // FIXME: should this really be checked in?
```

Comment:
I agree that this is far from ideal, but we need to define the workflow around how this should be maintained from a user experience standpoint.